src/main/kotlin/com/hibob/academy/service/PetService.kt

```diff
@@ -49,6 +50,30 @@ class PetService @Autowired constructor(private val petDao : PetDao){
         return petDao.adoptPet(petId,ownerId,companyId)
     }
 
+
+    fun adoptMultiplePets(listPetId: List<Long>, ownerId: Long, companyId: Long): Int {
+        if(listPetId.isEmpty()){
```

Comment:
if(listPetId.isEmpty()) and  if(companyId < 0) are check in both methods
You can extract a validation method for those and reuse